types/index.d.ts

```diff
@@ -35,19 +35,27 @@ export interface ShutterButton {
   hideShutterButton: boolean;
 }
 
+export interface SingleDocumentMode {
+  name?: string;
+  message?: string;
+}
+
+export interface MultipleDocumentMode {
+  name?: string;
+  message?: string;
+}
+
+export interface SegmentedDocumentMode {
+  name?: string;
+  message?: string;
+}
 
```

Comment:
Can't these 3 interfaces just be a single interface? Simply `DocumentMode` should be sufficient right?

Reply:
Yes, just `DocumentMode` is sufficient. Changed.